src/applications/ivc-champva/10-7959f-2/config/form.js

```diff
@@ -292,29 +298,71 @@ const formConfig = {
         page7: {
           path: 'upload-supporting-documents',
           title: 'Included files',
+          depends: formData => formData.sendPayment === 'Veteran',
+          uiSchema: {
+            ...titleUI({
+              title: 'Upload billing statements and supporting documents',
+              headerLevel: 2,
+            }),
+            'view:UploadDocuments': {
+              'ui:description': UploadDocumentsVeteran,
+            },
+            uploadSectionVeteran: fileUploadUI({
+              label: 'Upload file',
+              attachmentName: false,
+            }),
+          },
+          schema: {
+            type: 'object',
+            required: ['uploadSectionVeteran'],
+            properties: {
+              titleSchema,
+              'view:UploadDocuments': {
+                type: 'object',
+                properties: {},
+              },
+              uploadSectionVeteran: {
+                type: 'array',
+                minItems: 1,
+                items: {
+                  type: 'object',
+                  properties: {
+                    name: {
+                      type: 'string',
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+        page8: {
+          path: 'upload-supporting-documents-provider',
+          title: 'Included files',
+          depends: formData => formData.sendPayment === 'Provider',
           uiSchema: {
             ...titleUI({
               title: 'Upload billing statements and supporting documents',
               headerLevel: 2,
             }),
             'view:UploadDocuments': {
-              'ui:description': UploadDocuments,
+              'ui:description': UploadDocumentsProvider,
             },
-            uploadSection: fileUploadUI({
+            uploadSectionProvider: fileUploadUI({
               label: 'Upload file',
               attachmentName: false,
             }),
           },
           schema: {
             type: 'object',
-            required: ['uploadSection'],
+            required: ['uploadSectionProvider'],
             properties: {
               titleSchema,
               'view:UploadDocuments': {
                 type: 'object',
                 properties: {},
               },
-              uploadSection: {
+              uploadSectionProvider: {
                 type: 'array',
                 minItems: 1,
                 items: {
```

Comment:
Added new page here, the diff looks a bit wonky cause there's a lot of overlap with the previously existing file upload config. 

There are now two upload screens, `uploadSectionVeteran` and `uploadSectionProvider`